okhttp/src/main/java/okhttp3/internal/platform/android/AndroidQCertificateChainCleaner.kt

```diff
@@ -0,0 +1,37 @@
+package okhttp3.internal.platform.android
+
+import android.net.http.X509TrustManagerExtensions
+import okhttp3.internal.tls.CertificateChainCleaner
+import java.security.cert.Certificate
+import java.security.cert.CertificateException
+import java.security.cert.X509Certificate
+import javax.net.ssl.SSLPeerUnverifiedException
+import javax.net.ssl.X509TrustManager
+
+/**
+ * X509TrustManagerExtensions was added to Android in API 17 (Android 4.2, released in late 2012).
+ * This is the best way to get a clean chain on Android because it uses the same code as the TLS
```

Comment:
seems like the wrong docs

Reply:
It's just moved with the class

https://github.com/square/okhttp/blob/7cf508a30bb0656bb5c44ddafc16f7f8790c0187/okhttp/src/main/java/okhttp3/internal/platform/AndroidPlatform.kt#L172